utils.js

```diff
@@ -875,4 +876,16 @@ var drawRoundedLine = function(cr, x, y, width, height, isRoundLeft, isRoundRigh
     if (stroke != null)
         cr.setSource(stroke);
     cr.stroke();
-}
\ No newline at end of file
+}
+
+/**
+ * Check if an app exists in the system.
+ * Depends on "which" to find app in $PATH
+ */
+function checkExists(app) {
```

Comment:
Just a nitpick, but since this is now out of the context of adding the context menu items, we could change the name to be a bit more explicit... Something like checkIfCommandExists or something.

Reply:
I should have thought of that! Will do!

Reply:
I have been doing some more testing. Turns out 'which' isn't a good assumption either. I tried to test this on a very lightweight Arch install, and that doesn't have which in it! I'm going to do some more thinking on this and resubmit a better solution!

Reply:
I figured out how to use command -v to achieve the same result without major changes. That is a lower POSIX command implemented in bash, unlike which... which is a separate executable.